content/02.introduction.md

```diff
@@ -90,12 +90,16 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 
 #### Clinical Presentation of COVID-19
 
-Retrospective samples of COVID-19 patients described the clinical presentations of patients infected with SARS-CoV-2 which included lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
+A great diversity of symptom profiles has been observed for COVID-19, although fever and cough are very common.  [@doi:10.1056/NEJMoa2002032]
```

Comment:
```suggestion
A great diversity of symptom profiles has been observed for COVID-19, although fever and cough are very common and were initially reported on admission in 43.8% and 63.7%, respectively.  [@doi:10.1056/NEJMoa2002032]  Recent data from the New York City region have showed variable rates of fever as a presenting symptom, ranging from 30.7% to 77.1% of patients [@doi:10.1001/jama.2020.6775][@doi:10.1056/NEJMc2010419].  Of note, 20.4% of patients presented with a low oxygen saturation (<90%) [@doi:10.1001/jama.2020.6775], while others have noted cough (79.4%), fever (77.1%), and dyspnea (56.5%)as the most common presenting symptoms [@doi:10.1056/NEJMc2010419].  The variability shown in the presenting symptoms of infection by SARS-CoV-2 makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation.
```

Reply:
Maybe if we are going to provide exact percentages, we might have to tell more about the samples from each study.  The percentages would be different in a sample in China, Seattle, or New York, even if the age range was similar.  Do you think so?  I was intentionally being vague about exact numbers in order not to get into that, but I think it's fine to go into more detail too.

Reply:
That's fine with me.  I can remove the percentages so we don't get too specific.

Reply:
What do you think, @rando2 ?  Ordinarily I would think that more precise information was better.  But it is sometimes hard to decide.

Reply:
My thought process was that the presenting symptoms should not be very dependent on location.  Ie. the percentage of patients presenting with fever shouldn't change if you're in China vs. Seattle vs. NYC (which we show is not correct and it does vary).  Outcomes, on the other hand, would be very dependent on location, available care, drug regimens, etc. and the specific percentages for those would need to be further explained (ironically, these are all somewhat close between various regions and are very poor overall).

Reply:
Come to think of it, I suppose that sometimes, the differences in populations might be part of an interesting story.  For instance, the kidney pathology paper (#325) mentioned that acute kidney damage seemed to be more of a problem in New York than in Seattle or China, perhaps due to more preexisting problems like diabetes.  Francesca Dominici at Harvard also has a tech report linking COVID-19 deaths to air pollution levels, and of course there is a lot of justified concern about racial disparities.  But I don't know where, if anywhere, those macro-scale issues would fit into this particular paper.

Reply:
That gets very tricky and could really send us spiraling.  It's also incredibly hard to separate pre-existing conditions vs. racial disparities, as those are not mutually exclusive.  My institution's data (still under review, dragging on forever) suggests that it's more of the overall health picture of the patients rather than race that effect survival, but it's impossible to separate the two.  

I don't want this paper to get too dragged down with the clinical side (which is continually evolving) since it's more of an overarching review.

Reply:
That may be a good point -- maybe sometimes it's worse to say something simplistic than just to say nothing.  We won't be able to cite every existing COVID-19 paper anyway, since they are coming out faster than we could read them.

Reply:
I almost want to keep the whole clinical section broad and somewhat simple.  

- This is how they present (fever, cough, etc)
- These are the complications (DVT, PE, stroke, end-organ failure)
- These are the outcomes (we can get specific in % mortality or keep it broad and say something along the lines of "significant mortality has been shown across populations and pandemic hot spots, including Wuhan, Seattle, and NYC"

Reply:
That seems like a good structure -- @rando2, does this seem reasonable?

Reply:
This seems very reasonable to me! There is also always the option to add in more information as a new section down the road if it starts to seem like we need it. @scapone01 I edited your comment to move each sentence to a new line (which makes it a little easier to track edits), and I'll defer to @dziakj1 to commit it into the text!

Reply:
@rando2 Thanks so much!  I'll keep that in mind with my next set of edits.  I'm going to sit down and try to do them all at once so it's more organized and only one suggested change vs. multiple smaller ones.